content/blog/2025/10/2025-10-21-dinum.md

```diff
@@ -0,0 +1,82 @@
+
+This is particularly exciting news as it will hopefully set an example for other public sector organisations using Matrix to communicate, and there are many of them.
+
+<!-- more -->
+
+In France, Matrix is the basis for Tchap, the secure messaging system of choice for public officials. It was selected for its security, but also openness, transparency and sovereignty as it limits dependence on proprietary solutions from outside Europe.
+
+With this support, DINUM is committed to:
+
+* **participating in the governance of the protocol and enhancing its quality**, in conjunction with other user states (Sweden, Luxembourg, the Netherlands, Germany...) and the open source business ecosystem;  
+* **contributing to the development of the community versions of the various software components of the ecosystem**, used by many public services;  
+* ensuring that its actions are part of a **framework for sustainable investment in digital commons**.
+
+DINUM has been an active contributor to projects of the Matrix ecosystem for a while, including by providing improvements to the implementation of Sliding Sync, which optimises performance and user experience; as well as the development of a community version of a border gateway, which enhances the security of the communications between distinct organisations.
+
+Stéphanie Schaer, the Interministerial Director for Digital Affairs says: “With this partnership, we are choosing to make a long-term investment in a strategic digital commons for the benefit of the European ecosystem. Matrix is an essential building block for secure and sovereign communication services. This sponsorship reflects our conviction that the digital sovereignty of the State requires direct investment in the commons.”
+
+## Matrix for Public Sector
+
+This announcement comes a few days after the second edition of the [Matrix Conference](https://2025.matrix.org/). The Conference was held in Strasbourg (France), gathering about 300 attendees, amongst which more than a dozen public sector organisations, brought together by their use of Matrix as a sovereign, secure and resilient communication protocol.
+
+The Conference saw two full days of [public sector track](https://2025.matrix.org/schedule/), shedding light on the various exciting deployments of Matrix by these organisations. All the talks are already available for watching [here](https://www.youtube.com/playlist?list=PLl5dnxRMP1hUgnYEbpEsEEhIqY_KlO3NG), and will be soon published on the website, but you can find a list of the public sector related ones at the bottom of this post.
+
+We also took the opportunity of having many representatives of these public organisations present in person at the Conference to **officially launch a Matrix for Public Sector working group**. This is the very first working group of its kind that the Foundation sets up and already many other industries are asking for something similar.
```

Comment:
Is this supposed to mean a WG in the sense of https://matrix.org/foundation/working-groups/?

Reply:
```suggestion
We also took the opportunity of having many representatives of these public organisations present in person at the Conference to **officially launch a Matrix for Public Sector forum**. This is the very first group of its kind that the Foundation sets up and already many other industries are asking for something similar. The group will be proposed to the Governing Board for approval to potentially become an official [Working Group](https://matrix.org/foundation/working-groups/).
```